README.md

```diff
@@ -1,13 +1,18 @@
-# Determined Legacy Examples
+# Determined As-is Examples
 
-This Repository contains Determined examples that are no longer actively maintained by the determined team.
+This repository contains a variety of Determined examples that are not actively maintained by the Determined team.
 
 ## Tutorials
 
 | Example                                                       | Dataset          | Framework             |
 |:-------------------------------------------------------------:|:----------------:|:---------------------:|
 | [fashion\_mnist\_tf\_keras](tutorials/fashion_mnist_tf_keras) | Fashion MNIST    | TensorFlow (tf.keras) |
 
+## Blog posts
+| Example                                 | Description  |
+|:---------------------------------------:|:------------:|
```

Comment:
Should add more extend the Description with more whitespace and more `-` to follow the formatting of the rest of the tables.

Reply:
Done